lib/index.js

```diff
@@ -286,10 +286,23 @@ module.exports = {
 							return;
 						}
 
+						// handle "export" mode
+						const isExport = exportMode &&
```

Comment:
`isExportViolation`? :shrug:

Reply:
As mentioned in #29, I think we should drop this exclusive logic and just report both errors in the case of exporting an arrow expression.